seatunnel-connectors-v2/connector-file/connector-file-oss/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/oss/sink/OssFileSink.java

```diff
@@ -17,28 +17,82 @@
 
 package org.apache.seatunnel.connectors.seatunnel.file.oss.sink;
 
+import org.apache.seatunnel.shade.com.typesafe.config.Config;
+
+import org.apache.seatunnel.api.common.SeaTunnelAPIErrorCode;
 import org.apache.seatunnel.api.configuration.ReadonlyConfig;
+import org.apache.seatunnel.api.sink.DataSaveMode;
+import org.apache.seatunnel.api.sink.DefaultSaveModeHandler;
+import org.apache.seatunnel.api.sink.SaveModeHandler;
+import org.apache.seatunnel.api.sink.SchemaSaveMode;
+import org.apache.seatunnel.api.sink.SupportSaveMode;
+import org.apache.seatunnel.api.table.catalog.Catalog;
 import org.apache.seatunnel.api.table.catalog.CatalogTable;
+import org.apache.seatunnel.api.table.factory.CatalogFactory;
+import org.apache.seatunnel.common.config.CheckConfigUtil;
+import org.apache.seatunnel.common.config.CheckResult;
+import org.apache.seatunnel.common.constants.PluginType;
 import org.apache.seatunnel.connectors.seatunnel.file.config.FileSystemType;
+import org.apache.seatunnel.connectors.seatunnel.file.exception.FileConnectorException;
+import org.apache.seatunnel.connectors.seatunnel.file.oss.config.OssFileSinkOptions;
 import org.apache.seatunnel.connectors.seatunnel.file.oss.config.OssHadoopConf;
 import org.apache.seatunnel.connectors.seatunnel.file.sink.BaseMultipleTableFileSink;
 
 import java.util.Optional;
 
-public class OssFileSink extends BaseMultipleTableFileSink {
+import static org.apache.seatunnel.api.table.factory.FactoryUtil.discoverFactory;
+
+public class OssFileSink extends BaseMultipleTableFileSink implements SupportSaveMode {
 
     private final CatalogTable catalogTable;
 
+    private final ReadonlyConfig readonlyConfig;
+
     public OssFileSink(ReadonlyConfig readonlyConfig, CatalogTable catalogTable) {
         super(OssHadoopConf.buildWithConfig(readonlyConfig), readonlyConfig, catalogTable);
         this.catalogTable = catalogTable;
+        this.readonlyConfig = readonlyConfig;
+        Config pluginConfig = readonlyConfig.toConfig();
+        CheckResult result =
+                CheckConfigUtil.checkAllExists(
+                        pluginConfig,
+                        OssFileSinkOptions.FILE_PATH.key(),
+                        OssFileSinkOptions.BUCKET.key());
+        if (!result.isSuccess()) {
+            throw new FileConnectorException(
+                    SeaTunnelAPIErrorCode.CONFIG_VALIDATION_FAILED,
+                    String.format(
+                            "PluginName: %s, PluginType: %s, Message: %s",
+                            getPluginName(), PluginType.SINK, result.getMsg()));
+        }
     }
 
     @Override
     public String getPluginName() {
         return FileSystemType.OSS.getFileSystemPluginName();
     }
 
+    @Override
+    public Optional<SaveModeHandler> getSaveModeHandler() {
```

Comment:
why we should rewrite https://github.com/apache/seatunnel/blob/4a2f3fa91590db9c1f0b3ca081d466cb54b24dae/seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/sink/BaseMultipleTableFileSink.java#L132?

Reply:
I saw that S3FileSink has rewritten this method, let me confirm again.